lib/ontologies_linked_data/models/ontology_submission.rb

```diff
@@ -499,28 +501,8 @@ def generate_rdf(logger, reasoning: true)
           logger.error("Error sending data to triple store - #{e.response.code} #{e.class}: #{e.response.body}") if e.response&.body
           raise e
         end
-        version_info = extract_version()
```

Comment:
@mdorf and @alexskr - are either of you able to explain why this code for extracting submission versions and the `extract_version` method is deleted in this pull request? I'm interested to understand this.

Reply:
This code was moved to its own module: https://github.com/ncbo/ontologies_linked_data/blob/develop/lib/ontologies_linked_data/concerns/ontology_submissions/submission_metadata_extractor.rb#L7
It's still being called via the `extract_metadata` method:
https://github.com/ncbo/ontologies_linked_data/blob/develop/lib/ontologies_linked_data/models/ontology_submission.rb#L1008

Reply:
Yes, and in the future we can extend this module to extract more metadata (e.g description,...), when you want to get the full MOD metadata model. and as it a separate module, it will be easier to merge.